src/main/java/bio/terra/flight/dataset/create/DatasetCreateFlight.java

```diff
@@ -10,8 +11,8 @@
 
 public class DatasetCreateFlight extends Flight {
 
-    public DatasetCreateFlight(FlightMap inputParameters, Object applicationContext) {
-        super(inputParameters, applicationContext);
+    public DatasetCreateFlight(FlightMap inputParameters, Object applicationContext, AuthenticatedUser userInfo) {
+        super(inputParameters, applicationContext, userInfo);
```

Comment:
Stairway should not have to know about AuthenticatedUser class. All it needs is a string to match against ownerid.

Reply:
We want to have the owner id and the email if an admin wants a list of jobs they know who started them. i've abstracted out a UserInfo interface with just these 2 pieces of information that will be passed to stairway